modules/dreamview/frontend/src/components/StatusBar/Gears.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { observer } from 'mobx-react';
+
+@observer
+export default class Gears extends React.Component {
+
+  render() {
+    const {
+      currentGear,
+    } = this.props;
+
+    const gearAlphabet = currentGear.charAt(5);
```

Comment:
Have we handle cases if Dreamview doesn't receive batterySocPercentage and currentGear values?

Reply:
It has been modified.Thank u.